cfgov/v1/atomic_elements/organisms.py

```diff
@@ -1299,3 +1299,50 @@ class Meta:
         icon = 'image'
         label = 'CCT Data Snapshot'
         template = '_includes/organisms/data_snapshot.html'
+
+
+class ImageMapCoordinates(blocks.StructBlock):
+    left = blocks.FloatBlock(required=True)
```

Comment:
Do these need to be FloatBlocks, vs [IntegerBlocks](https://docs.wagtail.io/en/v2.8/topics/streamfield.html#integerblock) if they can range between 0-100? It'd be nice to add some validation here via use of `min_value` and `max_value`.

It might be an nicer UX to specify these via left, top, right, and bottom, instead of via width and height, at least if the user is entering these manually. I know you talked about possibly writing a script to make the data entry process easier.

Also, is it possible to add some validation that the coordinates make sense (via a custom `clean()` method on the block)? This would prevent the user from entering values that go off of the image, etc?

Reply:
@chosak Thanks for the suggestions about validation! I've added that in an upcoming commit. 

I wasn't sure if `coordinates` would be confusing as a label when two of the values are width and height, but I couldn't think of a more appropriate term. I went with width and height because those are likely to be the values exported from imaging tools & we'll probably recommend one of those tools even in a manual workflow (maybe an [online image map generator](https://www.zaneray.com/responsive-image-map/)).

I used FloatBlocks because the values in the existing explainers (and output by some imaging tools) are precise to two decimal places. Maybe that level of precision is unnecessary, though? I wonder if rounding would make any significant visual difference.